codecov_cli/services/staticanalysis/__init__.py

```diff
@@ -139,6 +139,31 @@ async def run_analysis_entrypoint(
         )
     else:
         logger.info("All files are already uploaded!")
+    try:
```

Comment:
for readability reasons, can we extract this and put it in a function, maybe "send_finish_signal()"?

Reply:
yes